Library/docs/source/learn/physical_digital_realities/anchoring.md

```diff
@@ -0,0 +1,357 @@
+       [](const AnchorCollectionResult& Result) {
+           // Process retrieved anchors within the area
+       }
+   );
+   ```
+
+* `GetAnchorsInSpace`: Retrieves all anchors associated with a specific CSP space.
+   This method is helpful when accessing all anchors tied to a defined space, such as a building or room. 
+
+   ```
+   AnchorSystem->GetAnchorsInSpace("spaceIdXYZ", 0, 10, [](const AnchorCollectionResult& Result) {
+       // Handle retrieved anchors in space
+   });
+   ```
+
+* `GetAnchorsByAssetCollectionId`: Retrieves anchors linked to a particular asset collection.
+   This query is useful when an anchor is associated with specific digital content stored in an asset collection.
+
+   ```
+   AnchorSystem->GetAnchorsByAssetCollectionId("assetCollectionIdABC", 0, 10, [](const AnchorCollectionResult& Result) {
+       // Process retrieved anchors by asset collection
+   });
+   ```
+
+### Updating Anchors
+
+In some scenarios, updating an anchor's attributes may be necessary, such as modifying its digital transform or reassigning it to a different space. Although CSP does not currently support direct updates to anchor attributes, developers can delete an existing anchor and create a new one with the updated parameters to achieve the same result.
+
+### Deleting Anchors
+
+CSP allows the deletion of anchors using the `DeleteAnchors` method. This operation enables developers to manage obsolete or irrelevant anchors effectively.
+
+**Code Example**
+
+```
+// The set of anchors that the client application wishes to delete.
+csp::common::Array<csp::common::String> AnchorIDsToDelete(1) = {"66338b7fcd13d3f163e38c25"};
+
+// The callback to respond to the completion of the anchor delete operation.                        
+csp::systems::NullResultCallback Callback = [](const NullResult& Result)
+{
+    if(Result.GetResultCode() == csp::systems::EResultCode::Success)
+    {
+        // Application logic goes here.
+    };
+};
+                          
+csp::systems::AnchorSystem* AnchorSystem = csp::systems::SystemsManager::Get().GetAnchorSystem();
+AnchorSystem->DeleteAnchors(AnchorIDsToDelete, Callback);
+```
+
+This code deletes a specified anchor and provides a callback to handle the result. If successful, the anchor will no longer be accessible or queryable in CSP.
+
+## Querying for Anchors
+
+Querying anchors in CSP allows developers to locate and manage anchors based on specific criteria. CSP provides methods to query anchors by location, space, or asset collection, enabling flexible and precise retrieval of anchor data.
+
+Anchors can be retrieved via CSP using three different methods in the Anchor System: `GetAnchorsInArea`, `GetAnchorsInSpace`, and `GetAnchorsByAssetCollectionId`.
+
+All of the anchor querying methods support paging. Each parameter allows the application to specify how many anchors to **Skip are** returning and a **Limit** to how many per invocation should be returned.
```

Comment:
[typo] `**Skip are**`  did you mean `**Skip** are`? I can't quite make sense of this sentence(?).  Is "are returning" meant to be there at all?

Reply:
You're right, not well worded at all! Replacing with

> The application can specify how many anchors to `Skip` returning (starting from the first anchor that would be returned by the query), and a `Limit` controlling the maximum number of anchors that the query can return.